core/frontend/src/components/version-chooser/VersionChooser.vue

```diff
@@ -297,70 +257,35 @@ export default Vue.extend({
       }
       return ''
     },
-    getLatestBeta() {
-      const ordered_list = this.sortVersions(
-        this.available_versions.remote.map((image) => image.tag)
-          .filter((tag) => this.isSemVer(tag) && tag.includes('beta')),
-      )
-      return ordered_list ? ordered_list[0] : undefined
-    },
-    getLatestStable() {
-      const ordered_list = this.sortVersions(
-        this.available_versions.remote.map((image) => image.tag)
-          .filter((tag) => this.isSemVer(tag) && !tag.includes('beta')),
-      )
-      return ordered_list ? ordered_list[0] : undefined
-    },
-    sortImages() {
-      this.available_versions = {
-        local: this.available_versions.local.sort(
-          (a: Version, b: Version) => Date.parse(b.last_modified) - Date.parse(a.last_modified),
-        ),
-        remote: this.available_versions.remote.sort(
-          (a: Version, b: Version) => Date.parse(b.last_modified) - Date.parse(a.last_modified),
-        ),
-        error: this.available_versions.error,
-      }
-    },
     async loadAvailableversions() {
-      this.available_versions = {
-        local: [],
-        remote: [],
-        error: null,
-      } as VersionsQuery
       this.loading_images = true
       this.error = null
-      await back_axios({
-        method: 'get',
-        url: `/version-chooser/v1.0/version/available/${this.selected_image}`,
-      }).then((response) => {
-        this.available_versions = response.data
-        this.error = response.data.error
-      }).catch(() => {
-        this.available_versions = {
-          local: [],
-          remote: [],
-          error: 'Unable to communicate with backend',
-        }
-      }).finally(() => {
-        this.loading_images = false
-        this.sortImages()
-        this.latest_beta = this.getLatestBeta()
-        this.latest_stable = this.getLatestStable()
-      })
+
+      await VCU.loadAvailableVersions(this.selected_image)
+        .then((versions_query) => {
+          this.available_versions = versions_query
+        })
+        .finally(() => {
+          this.loading_images = false
+          this.available_versions = VCU.sortImages(this.available_versions)
```

Comment:
is this lines required? they are [already sorted by loadAvailableVersions](https://github.com/bluerobotics/BlueOS-docker/pull/879/files#diff-50ff488bb99812e1025436fc7d2a5701694d2523a2174814071dc9cf81173245R113)

Reply:
Well, better safe than sorry. We may change the utils to be not sorted by default. We can also make it not sorted (but there is no advantage for that).